test/perf/test_performance.py

```diff
@@ -6,10 +6,10 @@
 from time import time
 from tempeh.configurations import models, datasets
 
-from fairlearn.post_processing import ThresholdOptimizer
-from fairlearn.post_processing.threshold_optimizer import DEMOGRAPHIC_PARITY
+from fairlearn.postprocessing import ThresholdOptimizer
+from fairlearn.postprocessing._threshold_optimizer import DEMOGRAPHIC_PARITY
```

Comment:
These look like leftovers from the file reorg :-)

Reply:
correct